bitrise.yml

```diff
@@ -167,9 +163,6 @@ workflows:
         inputs:
         - test_name: Flix Screenshot Test
         - search_pattern: "**test_result_**.xml**"
-    - deploy-to-bitrise-io@2:
```

Comment:
Why did this get removed?

Reply:
@lwise There were too many of these. They copy the artifacts to the `Artifacts` tab on the Bitrise report but there only needs to be one of these. There's one on L259